rhp/v4/merkle.go

```diff
@@ -106,42 +106,30 @@ func VerifySectorRootsProof(proof, sectorRoots []types.Hash256, numSectors, star
 	return rhp2.VerifySectorRangeProof(proof, sectorRoots, start, end, numSectors, root)
 }
 
-/*
-TODO: implement for RPC remove
-func convertActions(actions []ModifyAction) []rhp2.RPCWriteAction {
-	rhp2Actions := make([]rhp2.RPCWriteAction, len(actions))
-	for i, a := range actions {
-		switch a.Type {
-		case ActionSwap:
-			rhp2Actions[i] = rhp2.RPCWriteAction{
-				Type: rhp2.RPCWriteActionSwap,
-				A:    a.A,
-				B:    a.B,
-			}
-		case ActionTrim:
-			rhp2Actions[i] = rhp2.RPCWriteAction{
-				Type: rhp2.RPCWriteActionTrim,
-				A:    a.N,
-			}
-		case ActionUpdate:
-			rhp2Actions[i] = rhp2.RPCWriteAction{
-				Type: rhp2.RPCWriteActionUpdate,
-				A:    a.N,
-				Data: nil, // TODO
-			}
-		}
+func convertFreeActions(freed []uint64, numSectors uint64) []rhp2.RPCWriteAction {
```

Comment:
This is going to need to be rewritten after we delete RHP2 anyway. Can this be inlined and get rid of the generic rhp2.RPCWriteAction for good?

Reply:
Probably fine to keep this but I don't feel strongly about either approach

Reply:
I intend to go HAM on the Merkle stuff once the dust has fully settled